commit f095176610597c8e95481a400375a742f2e96a3a
Author: Karl Palsson <karlp@etactica.com>
Date:   Fri May 5 09:57:37 2017 +0000

    [344] Only link libanl when WITH_ADNS=yes
    
    Removes the hard link of libanl for Linux, leaving the protected link
    when WITH_ADNS is specified.
    
    Fixes: 433ee5c4d6852b507b43eae9c9a2c9028c6b48e5
    
    Signed-off-by: Karl Palsson <karlp@etactica.com>

diff --git a/config.mk b/config.mk
index 6e369c2..44639d2 100644
--- a/config.mk
+++ b/config.mk
@@ -159,10 +159,6 @@ ifeq ($(UNAME),QNX)
 	LIB_LIBS:=$(LIB_LIBS) -lsocket
 endif
 
-ifeq ($(UNAME),Linux)
-	BROKER_LIBS:=$(BROKER_LIBS) -lanl
-endif
-
 ifeq ($(WITH_WRAP),yes)
 	BROKER_LIBS:=$(BROKER_LIBS) -lwrap
 	BROKER_CFLAGS:=$(BROKER_CFLAGS) -DWITH_WRAP
